iceberg/iceberg-handler/src/test/queries/positive/iceberg_initial_default.q

```diff
@@ -0,0 +1,100 @@
+CREATE TABLE ice_parq (
+  id INT)
+STORED BY ICEBERG  stored as parquet
+TBLPROPERTIES ('format-version'='3');
+
+INSERT INTO ice_parq (id) VALUES (1);
+
+ALTER TABLE ice_parq ADD COLUMNS (point STRUCT<x:INT, y:INT> DEFAULT 'x:100,y:99',
```

Comment:
How is it different from https://github.com/apache/hive/pull/6120/files#diff-1d416e605ef70101f82b2d245c762b7c1ec5d89bdf7d82f5fac9c18211bb6f18? 
Can't it be extended?